app/views/insured/product_shoppings/show.html.slim

```diff
@@ -437,7 +447,7 @@ javascript:
                     </a>
                   </div>
                   <div class="col-md-5 col-sm-5 col-xs-5 pl-0 pr-0">
-                    <a class="btn btn-default btn-right plan-select select" href="/insured/plan_shoppings/#{@context.hbx_enrollment.id.to_s}/thankyou?change_plan=&coverage_kind=${result.product.kind}&enrollment_kind=&market_kind=${result.product.benefit_market_kind.replace('aca_','')}&plan_id=${result.product._id}">Select Plan</a>
```

Comment:
if this is used for both old and new flow, can you use resource registry settings?

Reply:
this file is only used in the new flow, no need to add RR settings for this line